OSPO-101/module1/README.md

```diff
@@ -368,3 +368,4 @@ Standards and open source move at different paces, and work on interoperability
 
 Additionally, by supporting standards, the open source ecosystem gets the ability to interoperate even with closed source software, which provides businesses with more choices and more flexibility.
 
+In the other direction, standard writing comitees have the ability to validate thier drafts via open source projects to fastly iterate on viability or corner cases. For example, GCC and Clang projects frequently provide implementation of standard-draft new language features to provide quick feedback-loop to standard bodies defining those languages.
```

Comment:
Thanks for bringing the standard writing committee's use case into the course! I think is a nice additional perspective to include